lua/starfall/libs_cl/render.lua

```diff
@@ -868,6 +887,22 @@ function render_library.setRGBA(r, g, b, a)
 	surface.SetTextColor(r, g, b, a)
 end
 
+--- Gets the drawing tint.
+-- @return Color Color (in 0-255 range)
```

Comment:
Just a nitpick, but color objects do not wrap values to `0-255` by themselves, so the range is technically undefined. Would be more accurate to just specify that it's just multiplied by `255`. Same for `render.setTint`.